bot/code_review_bot/report/base.py

```diff
@@ -146,6 +146,9 @@ def pluralize(word, nb):
         else:
             comment = ""
 
+        if any(revision.contains(issue) for issue in issues):
```

Comment:
That's actually the opposite test: you need to add the sentence, when the revision does not contain every publishable issue:

```python
if not all(revision.contains(issue) for issue in issues if issue.is_publishable):
 ...
```